spec/features/staff/users_can_create_a_budget_spec.rb

```diff
@@ -1,40 +1,44 @@
 RSpec.describe "Users can create a budget" do
-  before { authenticate!(user: user) }
+  context "when the user is n
```

Comment:
I realise this change has been superseded by another but I am not 100% happy with this malformed code being in a commit 😳

Reply:
Yes, gosh what have I managed to get myself into here!

Reply:
I think I've got rid of this. It all went wrong after conflict resolution with the parent pull request. I dropped the shared origin commit which included this issue and problem solved.